python-checks/src/test/resources/checks/verifiedSslTlsCertificateCheck.py

```diff
@@ -0,0 +1,282 @@
+  #                                                        ^^^^^^^^
+  requests.request(verify=False, method='GET', url='https://example.domain') # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                       ^^^^^
+  kargs1 = {'verify': False} # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                   ^^^^^
+  requests.request('GET', 'https://example.domain', **kargs1)
+  kargs2 = {'method': 'GET', 'url': 'https://example.domain', 'verify': False} # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                                                                     ^^^^^
+  requests.request(**kargs2)
+
+  requests.get('https://example.domain', verify=False) # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                                             ^^^^^
+  requests.head('https://example.domain', verify=False) # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                                              ^^^^^
+  requests.post('https://example.domain', verify=False) # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                                              ^^^^^
+  requests.put('https://example.domain', verify=False) # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                                             ^^^^^
+  requests.delete('https://example.domain', verify=False) # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                                                ^^^^^
+  requests.patch('https://example.domain', verify=False) # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                                               ^^^^^
+  requests.options('https://example.domain', verify=False) # Noncompliant {{Disabling certificate verification is dangerous.}}
+  #                                                 ^^^^^
+
+  requests.request(method='GET', url='https://example.domain') # Compliant
+  requests.request(method='GET', url='https://example.domain', verify=True) # Compliant
+  requests.request('GET', 'https://example.domain', verify='/path/to/CAbundle') # Compliant
+  requests.request(verify=True, method='GET', url='https://example.domain') # Compliant
+  kargs = {'verify': True}
+  requests.request('GET', 'https://example.domain', **kargs) # Compliant
+  kargs = {'method': 'GET', 'url': 'https://example.domain', 'verify': True}
+  requests.request(**kargs) # Compliant
+
+  requests.head(url='https://example.domain') # Compliant
+  requests.get(url='https://example.domain') # Compliant
+  requests.post(url='https://example.domain') # Compliant
+  requests.put(url='https://example.domain') # Compliant
+  requests.patch(url='https://example.domain') # Compliant
+  requests.delete(url='https://example.domain') # Compliant
+  requests.options(url='https://example.domain') # Compliant
```

Comment:
I don't think you need to be exhaustive here by listing all the API, probably only one would be enough 

```python
requests.head(url='https://example.domain') # Compliant
```